### PR TITLE
Update NVIDIA documentation to isolate install even further

### PR DESCRIPTION
**Pull request by puneetse**
This PR:

- Isolates the NVIDIA installed files even further into `/opt` and `/etc`. This adds steps to the process but avoids files being removed with CL updates and breaking the NVIDIA driver installation. It does this by

   - Add config to search Xorg modules under /opt
   - Add config to move libglvnd files under /etc (reference[ #1](https://github.com/NVIDIA/libglvnd/blob/55fad54ac79f5cf5d2a7f6e61596a26e0118daba/src/EGL/Makefile.am#L54) [#2](https://fedora.pkgs.org/30/fedora-x86_64/libglvnd-egl-1.1.0-4.gitf92208b.fc30.x86_64.rpm.html))

- Updates the `swupd verify --fix ` command to the new `swupd repair`

- Adds an explicit warning about the well-known `libGL.so` conflict between distro provided packaged, like mesa and NVIDIA.

### CODE REVIEW

**Comment by puneetse** [comment]
@lebensterben FYI

File: source/clear-linux/tutorials/nvidia.rst [hunk 1]
```diff
@@ -107,25 +112,45 @@ needs to be disabled before installation can continue.
...
+   .. code-block:: bash
+
+      sudo ldconfig
+
+#. Create a Xorg configuration file to search for modules under
+   :file:`/opt/nvidia` in addition to the default path.
+
+   .. code-block:: bash
+
+      sudo tee /etc/X11/xorg.conf.d/nvidia-files-opt.conf > /dev/null <<'EOF'
```
Comment (bwarden) [inline source/clear-linux/tutorials/nvidia.rst]: Add a mkdir before this

File: source/clear-linux/tutorials/nvidia.rst [hunk 2]
```diff
@@ -185,6 +215,7 @@ Install the NVIDIA Drivers
 
    The NVIDIA software places some files under the :file:`/usr` subdirectory
    which are not managed by |CL| and conflict with the |CL| stateless design.
+
    Although a limited version of :command:`swupd repair` is ran above,
```
Comment (bwarden) [inline source/clear-linux/tutorials/nvidia.rst]:
```suggestion
   Although a limited version of :command:`swupd repair` is run above,
```

**Comment by puneetse** [comment]
Thanks @bwarden, I've pushed those changes.

**Comment by puneetse** [comment]
@mvincerx can you review and merge if no other blockers?

**Comment by mvincerx** [comment]
Doing so already.

**Comment by mvincerx** [comment]
LGTM. Clean build.

**Pull request merged**